### PR TITLE
workflows: Update actions/checkout to v3

### PR DESCRIPTION
**Pull request by Sushrut1101**

### CODE REVIEW

**Comment by Sushrut1101** [comment]
Signed!

> [CLA assistant check](https://cla-assistant.io/airbytehq/airbyte?pullRequest=17764) All committers have signed the CLA.

**Comment by marcosmarxm** [comment]
@supertopher can you review this community contribution?

File: .github/workflows/commands-for-testing-tool.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
             ${{ secrets.DAVINCHIA_PAT }}
       - name: Get PR repo and ref
         id: getref
-        run: |
+       AirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyteAirbyte
```
Comment (marcosmarxm) [inline .github/workflows/commands-for-testing-tool.yml:27]: what is this?

**Comment by marcosmarxm** [comment]
@Sushrut1101 do you want to merge this? Can you add a complete description of what of you're trying to achieve changing action v2 to v3?

File: .github/workflows/commands-for-testing-tool.yml [hunk 2]
```diff
@@ -13,8 +13,8 @@ jobs:
       comment-id: ${{ steps.comment-info.outputs.comment-id }}
       command: ${{ steps.regex.outputs.first_match }}
     steps:
-      - name: Checkout Airbyte
-        uses: actions/checkout@v2
+      - name: Checkout AirbyteAirbyte
```
Comment (marcosmarxm) [inline .github/workflows/commands-for-testing-tool.yml:16]: ?

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

**Comment by supertopher** [comment]
Agree with the above comments.  Please articulate what you want to achieve with this change.  I'm not opposed to upgrading, but without a reason this feels like a risk without a reward.  Version bumps may have unintended effects especially considering the number of places this action runs.

Thank you for the contribution.  look forward to hearing from you

**Review by supertopher — CHANGES_REQUESTED** [review]
marking as requested changes to clear my notification

**Review by davinchia — CHANGES_REQUESTED** [review]
I agree with the other comments. Marking as request changes to clear my notifications too.

File: .github/workflows/commands-for-testing-tool.yml (hunk 2 above)
Comment (Sushrut1101) [reply .github/workflows/commands-for-testing-tool.yml:16]: Uhh, looks like sth got messed up 😬

**Comment by Sushrut1101** [comment]
Will do a force push, looks like I messed up sth 😬

**Comment by marcosmarxm** [comment]
> Will do a force push, looks like I messed up sth 😬

@Sushrut1101 please check commentns from Topher and Davin.

**Comment by marcosmarxm** [comment]
Hello :wave:, first thank you for this amazing contribution. 

We really appreciate the effort you've made to improve the project.
We ask you patience for the code review. Last month our team was focused on Hacktoberfest event and that probably left some PR without the proper feedback. And this week, due to the Thanksgiving US Holiday, most our team is out of office with their families. Another important piece of information why code won't be merge this week is: as a safety measure the core team has decided to freeze merging code to main branch to keep the release stable. Next week we'll return to you with the proper code review and update the status of your contribution.

If you have any questions feel free to send me a message in Slack!
Thanks!

**Pull request closed without merging**